tasks-by-users/AbbasShaikh0304/01-E-Commerce website/Style.css

```diff
@@ -0,0 +1,601 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: "Poppins", sans-serif;
+}
+.navbar {
+  display: flex;
+  align-items: center;
+  padding: 20px;
+}
+nav {
+  flex: 1;
+  text-align: right;
```

Comment:
here @AbbasShaikh0304

Reply:
instead of adding here and there, would you please suggest me the lines? it will be easy and helpfull for me to commit the changes. @Tauqeer-Ahmed-99